index.js

```diff
@@ -38,18 +65,32 @@
  * And returns the first and last name concatenated together (i.e. 'Bob Evans').
  */
 
+function getFullName (object) {
+	return object.firstName + ' ' + object.lastName;
+};
```

Comment:
The logic here is completely right. Generally, it is a good idea to name the parameter something more about what the Object represents, rather than the fact that is an Object:

```js
function getFullName(person) {
    return person.firstName + ' ' + person.lastName;
}
```